firedrake/slate/slac/compiler.py

```diff
@@ -479,7 +479,7 @@ def facet_integral_loop(cxt_kernel, builder, coordsym, cellfacetsym,
         tensor = eigen_tensor(exp, t, index)
 
         if kinfo.oriented:
-            clist.append(cell_orientations)
+            clist.insert(0, cell_orientations)
```

Comment:
My previous testing didn't catch that the assembly kernels take the `cell_orientation` arg before any coefficient args. This fixes it up and makes sure the args are passed to the assembly kernels in the right order.